### PR TITLE
kubeedge: added cross build job for edge

### PR DESCRIPTION
**Pull request by sids-b**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Added cross build job for edge / To see compatibility with different H/W architectures

### CODE REVIEW

File: edge/Makefile [hunk 1]
```diff
@@ -28,3 +28,12 @@ integration_test:
 .PHONY: clean
 clean:
 	bash -x hack/clean.sh
+
+.PHONY: cross_build
+cross_build:
+	export GOARCH=arm
```
Comment (m1093782566) [inline edge/Makefile]: So the default arch is `amd` right? What about
```
for arch in ("amd", "arm"); then
do
 export  GOARCH=$arch
 go build
done
```

Comment (sids-b) [reply edge/Makefile]: How about we keep this for loop for all other h/w platforms we support and amd we keep seperate as a basic one ??

Comment (m1093782566) [reply edge/Makefile]: OK, no problem. Either way it ok to me :)

Comment (m1093782566) [reply edge/Makefile]: Just for clarification :)

Comment (rohitsardesai83) [reply edge/Makefile]: See this : http://www.codershaven.com/multi-platform-makefile-for-go/

can we try to do something similar ?

**Pull request merged**